staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types_jsonschema.go

```diff
@@ -122,6 +122,61 @@ type JSONSchemaProps struct {
 	//      Atomic maps will be entirely replaced when updated.
 	// +optional
 	XMapType *string
+
+	// x-kubernetes-validations -kubernetes-validations describes a list of validation rules written in an expression language.
+	// This field is an alpha-level. Using this field requires the feature gate `CustomResourceValidationExpressions` to be enabled.
+	XValidations ValidationRules
+}
+
+// ValidationRules describes a list of validation rules written in an expression language.
```

Comment:
nit: the "an" sounds strange. There is only one.

Reply:
Will go away when we switch to "the CEL expression language".